addons/dark-www/addon.json

```
@@ -2297,6 +2297,25 @@
       "name": "\"Experimental\" Dark",
       "id": "experimentalDark",
       "description": "A dark theme by Maximouse based on a userstyle. Not experimental",
+      "values": {
+        "page": "#202020",
+        "navbar": "#855cd6",
+        "box": "#282828",
+        "gray": "#333333",
+        "blue": "#252c37",
+        "input": "#202020",
+        "button": "#855cd6",
+        "link": "#ccb3ff",
+        "footer": "#333333",
+        "border": "#606060",
+        "darkBanners": "darker",
+        "darkForumCode": true
+      }
+    },
+    {
+      "name": "\"Experimental\" Dark (blue)",
+      "id": "experimentalDark",
+      "description": "A dark theme by Maximouse based on a userstyle. Not experimental",
```

```suggestion
      "description": "\"Experimental\" Dark with a blue accent color",
```
(based on the description for the blue version of 3.Darker)